src/validation_workflow/validation_args.py

```diff
@@ -0,0 +1,74 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+import logging
+
+from validation_workflow.validation_rpm import ValidationRpm
+from validation_workflow.validation_tar import ValidationTar
+from validation_workflow.validation_yum import ValidationYum
+
+
+class ValidationArgs:
+    SUPPORTED_DISTRIBUTIONS = [
+        "tar",
+        "yum",
+        "rpm"
+    ]
+    SUPPORTED_PLATFORMS = ["linux"]
+    SUPPORTED_ARCHITECTURES = [
+        "x64",
+        "arm64",
+    ]
+    DISTRIBUTION_MAP = {
+        "tar": ValidationTar,
+        "yum": ValidationYum,
+        "rpm": ValidationRpm
+    }
+
+    version: str
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Download Artifacts.")
```

Comment:
Can you make the description more generic as the same framework is going to be used for multiple other platforms?